clients/drcachesim/tests/CMakeLists.txt

```diff
@@ -62,3 +64,6 @@ target_link_libraries(analyzer_separate drmemtrace_analyzer drmemtrace_histogram
 if (ZLIB_FOUND)
   target_link_libraries(analyzer_separate ${ZLIB_LIBRARIES})
 endif ()
+if (liblz4)
+  target_link_libraries(analyzer_separate lz4)
```

Comment:
I was initially confused about how analyzer_separate is related to drmemtrace_proj, then I saw the comment in clients/drcachesim/CMakelists.txt:

```
# We have a companion test built using a separate --build-and-test CMake project in
# tests/analyzer_separate.cpp to better test 3rd-party usage.
```
I suppose drmemtrace_proj is the test being referred to above right?

Maybe add more details to the PR description as to how they're connected.

Reply:
Done, and updated title and changed the code completely as it turns out we need just the search and not the link.